support/sparse_mmap/src/unix.rs

```diff
@@ -95,6 +95,14 @@ impl SparseMapping {
     /// The range will be aligned to the largest system page size that's smaller
     /// or equal to `len`.
     pub fn new(len: usize) -> Result<Self, Error> {
+        Self::new_at(len, None)
+    }
+
+    /// Reserves a sparse mapping range with the given size at specific GPA.
+    ///
+    /// The range will be aligned to the largest system page size that's smaller
+    /// or equal to `len`.
+    pub fn new_at(len: usize, fixed_addr: Option<u64>) -> Result<Self, Error> {
```

Comment:
Why do we need this?

Reply:
This is still a merge leftover, on my TODO list to review it.